.travis.yml

```diff
@@ -23,41 +21,6 @@ stages:
 # Each job will use the previous job's `stage` tag if not specified.
 jobs:
   include:
-  # patch formatting checks
-  - name: "Check patch formatting"
-    stage: check
-    os: linux
-    language: python
-    python: 3.7
-    before_install: bash ./scripts/ci/install.sh
-    install:
-    script: ./scripts/ci/check_patch.sh
-  # test jobs
-  - name: "3.7 on Windows"
-    stage: test
-    os: windows           # Windows 10.0.17134 N/A Build 17134
-    language: shell       # 'language: python' is an error on Travis CI Windows
-    env:
-      - PYTHON_VERSION=3.7.5
-      - PATH=/c/Python37:/c/Python37/Scripts:$PATH
-  - name: "3.8 on Windows"
-    os: windows           # Windows 10.0.17134 N/A Build 17134
-    language: shell       # 'language: python' is an error on Travis CI Windows
-    env:
-      - PYTHON_VERSION=3.8.0
-      - PATH=/c/Python38:/c/Python38/Scripts:$PATH
-  - name: "3.6 on Linux"
-    language: python
-    python: 3.6
-  - name: "3.7 on Linux"
-    language: python
-    python: 3.7
-  - name: "3.8 on Linux"
-    language: python
-    python: 3.8
-  - name: "3.7 on Mac"
-    os: osx
-    osx_image: xcode11.3
```

Comment:
Maybe we should leave travis builds until we make all tests on GHA work?

Reply:
Let's not remove that till we are running all of the tests, we might introduce regressions and not even know it.

Reply:
@skshetry after talk with @efiop we decided that we could skip those builds, as long as we keep the original Issue open and focus on fixing leftover tests.

Reply:
Up to you, but I'll still prefer to use Travis as a source of _truth_ for CI, as it might still take a week or more for that to complete. Anyway, the package builds will still be running there.

Reply:
@skshetry you are right, that would be ideal, but we are starting to have more issues with travis and fixing both seems like a bit too much. If we can make gha work reasonably well, then we could just get rid of travis and focus on final gha adjustments.

Reply:
@efiop, I'm just stating my preference, but if it takes too much effort, let's get rid of it then.